plugins/uniques.js

```diff
@@ -487,16 +647,77 @@ window.plugin.uniques.setupPortalsList = function() {
 					ev.preventDefault();
 					return false;
 				}, false);
+			},
 		},
-	});
-}
+//---------------------------------------------------------------------------
+		{title: "S",
```

Comment:
`title` better move to next line, indented

Reply:
done